tests/map/options.test.ts

```diff
@@ -61,4 +64,63 @@ describe('MapOptions', () => {
             assert(max.innerHTML === 'max:');
         }
     });
+
+    it('selected property changes when changed', () => {
+        const root = document.createElement('div');
+        const options = new MapOptions(root, DUMMY_PROPERTIES, DUMMY_CALLBACK);
+
+        checkPropertySelect(options.x, 'x');
+        checkPropertySelect(options.y, 'y');
+        checkPropertySelect(options.z, 'z');
+        checkPropertySelect(options.color, 'color');
+
+        function checkPropertySelect(axisOptions: AxisOptions, axisName: string) {
+            const selectElement = getByID<HTMLSelectElement>(`chsp-${axisName}`);
+            selectElement.value = 'second';
```

Comment:
We want to check that the value changes, so we need to check that the new & previous value are different, with something like this:

```suggestion
            assert(selectElement.value !== 'second');
            assert(axisOptions.property.value !== 'second');
            selectElement.value = 'second';
```